### PR TITLE
Bug start directory

### PR DESCRIPTION
**Pull request by joseph-flinn**
`start_directory` was not being passed through when creating a new session.

Issue: https://github.com/tmux-python/tmuxp/issues/631

### CODE REVIEW

File: tmuxp/__about__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 __title__ = 'tmuxp'
 __package_name__ = 'tmuxp'
-__version__ = '1.5.7post0'
+__version__ = '1.5.7post1'
```
Comment (tony) [inline tmuxp/__about__.py:3]: @joseph-flinn Can you make this 1.5.8?

```suggestion
__version__ = '1.5.8'
```

**Review by tony — APPROVED** [review]
@joseph-flinn Make it 1.5.8 and update CHANGES - I will publish it now :blush:

File: tmuxp/workspacebuilder.py [hunk 2]
```diff
@@ -137,9 +137,15 @@ def build(self, session=None):
...
                 )
             else:
-                session = self.server.new_session(
-                    session_name=self.sconf['session_name']
-                )
+                if 'start_directory' in self.sconf:
+                    session = self.server.new_session(
+                        session_name=self.sconf['session_name'],
+                        start_directory=self.sconf['start_directory'],
+                    )
```
Comment (tony) [inline tmuxp/workspacebuilder.py:144]: It's late so I don't want to dredge this up, do we want/need a test for this?

Comment (tony) [reply tmuxp/workspacebuilder.py:144]: I'm happy merging without a test, I can write one later if we need it

File: tmuxp/__about__.py (hunk 1 above)
Comment (joseph-flinn) [reply tmuxp/__about__.py:3]: yep

File: tmuxp/workspacebuilder.py (hunk 2 above)
Comment (joseph-flinn) [reply tmuxp/workspacebuilder.py:144]: No worries. It would probably be a good idea to write a test just so we have it

**Comment by tony** [comment]
@joseph-flinn v1.5.8 live

**Pull request merged**